src/components/Cluster/ClusterDetail/ClusterApps.tsx

```diff
@@ -322,25 +317,16 @@ const ClusterApps: React.FC<IClusterAppsProps> = ({
         )}
       </div>
 
-      {appToDisplay &&
-        (user?.type === LoggedInUserTypes.MAPI ? (
-          <AppDetailsModalMAPI
-            appName={appToDisplay.metadata.name}
-            catalog={appToDisplay.spec.catalog}
-            clusterId={clusterId}
-            onClose={hideAppModal}
-            visible={detailsModalIsVisible}
-          />
-        ) : (
-          <AppDetailsModal
-            // Instead of just assigning the selected app to the state of this component,
-            // this ensures any updates to the apps continue to flow down into the modal.
-            app={appToDisplay}
-            clusterId={clusterId}
-            onClose={hideAppModal}
-            visible={detailsModalIsVisible}
-          />
-        ))}
+      {appToDisplay && (
```

Comment:
No longer need for this logic. `<ClusterDetailApps />` will be rendered instead of `<ClusterApps />` when the user is logged in via SSO